lib/cli-engine/formatters/stylish.js

```diff
@@ -60,21 +104,19 @@ module.exports = function(results) {
                     messageType = chalk.yellow("warning");
                 }
 
-                return [
-                    "",
-                    message.line || 0,
-                    message.column || 0,
+                const row = [
+                    (message.line || 0).toString(),
+                    (message.column || 0).toString(),
                     messageType,
-                    message.message.replace(/([^ ])\.$/u, "$1"),
-                    chalk.dim(message.ruleId || "")
+                    message.message.replace(/([^ ])\.$/u, "$1")
                 ];
-            }),
-            {
-                align: ["", "r", "l"],
-                stringLength(str) {
-                    return stripAnsi(str).length;
+
+                if (message.ruleId) {
+                    row.push(chalk.dim(message.ruleId));
                 }
-            }
+
+                return row;
+            })
         ).split("\n").map(el => el.replace(/(\d+)\s+(\d+)/u, (m, p1, p2) => chalk.dim(`${p1}:${p2}`))).join("\n")}\n\n`;
```

Comment:
> `.split("\n")`

The last thing `table()` does is effectively `.join("\n")`...  which is then immediately reverted by this `.split("\n")`. Remove both and the code will be both shorter & less runtime-heavy. ✂️

Reply:
Yeah, I am planning to in-line the table code even more and switch this logic over to pure string concatenation for performance reasons.